modules/tooling-support/src/main/java/org/mule/runtime/module/tooling/internal/artifact/value/ValueProviderExecutor.java

```diff
@@ -180,4 +181,18 @@ private ValueProviderMediator createValueProviderMediator(ParameterizedModel par
     return empty();
   }
 
+  private Set<Value> getValues(String providerName,
+                               ValueProviderMediator<?> valueProviderMediator,
+                               ParameterValueResolver parameterValueResolver,
+                               ExtensionResolvingContext context)
+      throws org.mule.sdk.api.values.ValueResolvingException {
```

Comment:
use qualified import. Also, remember that the legacy Exception extends this one, so ou can use them interchangealy.